### PR TITLE
Editorial and typo fixes

### PR DESCRIPTION
**Pull request by DariuszDepta**

### CODE REVIEW

File: src/executor.rs [hunk 1]
```diff
@@ -29,7 +29,7 @@ impl AppResponse {
 
     /// Checks if there is an Event that is a super-set of this.
     ///
-    /// It has the same type, and all compare.attributes are included in it as well.
+    /// It has the same type, and all compare attributes are included in it as well.
```
Comment (jawoznia) [inline src/executor.rs:32]:
```suggestion
    /// It has the same type, and all compared attributes are included in it as well.
```

File: src/app_builder.rs [hunk 2]
```diff
@@ -294,7 +294,7 @@ where
         }
     }
 
-    /// Overwrites the default custom messages handler.
+    /// Overwrites the default custom messages' handler.
```
Comment (jawoznia) [inline src/app_builder.rs:297]: Is this ' a typo?

**Review by jawoznia — APPROVED** [review]
Minor fixes and otherwise LGTM

File: src/app_builder.rs (hunk 2 above)
Comment (DariuszDepta) [reply src/app_builder.rs]: Changed to: *Overwrites the default handler for custom messages.*

**Pull request merged**